v3/YALLToken.md

```diff
@@ -1,4 +1,4 @@
-# CoinToken
+# YALLToken
 
 ERC20 compatible token with the following functionlaity:
```

Comment:
ERC20 compatible token with the following functionality:
* additional commission, the amount of which is determined by `YALLGovernance`;
* Gas Station Network support;
* token transfer can only be performed between verified users (
user status is defined in `YALLDistributor`) and the whitelisted  contracts (determined in token contract);
* token balance can be restored by `YALLDistributor` contract.